js/src/forum/Components/EventTeaser.js

```diff
@@ -47,28 +46,24 @@ export default class EventTeaser extends Modal {
                 <div>
                   <h3 className="fa-pull-left App-titleControl App-titleControl--text" style="margin-right:1em">{this.title()}</h3>
                 </div>
-                <div className="fa-pull-right" >{avatar(this.props.event.user())}</div>
+                <div className="fa-pull-right" >{avatar(this.attrs.event.user())}</div>
                 <div style="clear:both">
               </div>
             </div>
 
             {this.alertAttrs ? <div className="Modal-alert">{Alert.component(this.alertAttrs)}</div> : ''}
             <div className="Modal-body">
-              <EventFragment event={this.props.event} />
+              <EventFragment event={this.attrs.event} />
               <p>
                 {LinkButton.component({
-                  children:"More details about this event",
-                  href: app.route.advevent(this.props.event),
-                })}
+                  href: app.route.advevent(this.attrs.event),
+                }, "More details about this event" )}
```

Comment:
Ideally  this is a translation call instead of hard coded string ( i know i started it.. does not need to be part of this PR)

Reply:
Had to put the string here but yeah, could be better with the internationalization function.